server/store/crypt_test.go

```diff
@@ -0,0 +1,89 @@
+package store
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPad(t *testing.T) {
+	assert := assert.New(t)
+	resp := pad(make([]byte, 3))
+	assert.Equal([]byte{0x0, 0x0, 0x0, 0xd, 0xd, 0xd, 0xd, 0xd, 0xd, 0xd, 0xd, 0xd, 0xd, 0xd, 0xd, 0xd}, resp)
+}
+
+func TestUnpad(t *testing.T) {
+	assert := assert.New(t)
+	resp, err := unpad(make([]byte, 1))
+	assert.Nil(err)
+	assert.Equal([]byte{0x0}, resp)
+}
+
+func TestEncrypt(t *testing.T) {
```

Comment:
This is not testing the result.

Also I would like to get a test that proof the symmetry between the two functions something that verify that encrypts and decrypts works well together.

Reply:
updated the test case. Let me know if any further changes need to be done

Reply:
@jespino updated the test case. Let me know if any further changes need to be done